server/index-sv.js

```diff
@@ -1,21 +1,30 @@
 const express = require('express');
 const db = require('../database/index-db.js').connection;
 const app = express();
+const cors = require('cors');
 
-const PORT = 1234;
+const PORT = 8002;
+
+app.use(cors({
+  origin: 'http://localhost:8000'
+}));
 
 app.use(express.json());
 app.use(express.urlencoded());
 app.use(express.static('public'))
+app.use('/bundle', express.static('public/bundle.js'));
 
-app.get('/:id', (req, res) => {
+app.get('/products/:id', (req, res) => {
```

Comment:
its fine that you changed this route, but it wasn't necessary for the proxy server to work.  this route is independent from the proxy server